client_delivery/shared/src/commonMain/kotlin/presentation/login/composable/CustomBottomSheet.kt

```diff
@@ -0,0 +1,119 @@
+package presentation.login.composable
+
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.beepbeep.designSystem.ui.composable.modifier.noRippleEffect
+import com.beepbeep.designSystem.ui.theme.Theme
+
+
+@Composable
+fun CustomBottomSheet(
+    sheetContent: @Composable () -> Unit,
+    sheetState: BottomSheetState,
+    onBackGroundClicked: () -> Unit = {},
+    sheetBackgroundColor: Color = Theme.colors.background,
+    sheetShape: RoundedCornerShape = RoundedCornerShape(
+        topStart = Theme.radius.medium,
+        topEnd = Theme.radius.medium
+    ),
+    content: @Composable () -> Unit
+) {
+
+    Box(modifier = Modifier.fillMaxSize()) {
+
+        content()
+
+        AnimatedVisibility(
+            visible = sheetState.isVisible,
+            enter = fadeIn(),
+            exit = fadeOut(),
+        ) {
+            Box(Modifier.fillMaxSize().background(Color.Black.copy(alpha = .4f)))
+        }
+
+        AnimatedVisibility(
+            visible = sheetState.isVisible,
+            enter = slideInVertically(
+                initialOffsetY = { it },
+                animationSpec = tween(300)
+            ),
+            exit = slideOutVertically(
+                targetOffsetY = { it },
+                animationSpec = tween(300)
+            ),
+        ) {
+            SheetContent(sheetShape, sheetBackgroundColor, { onBackGroundClicked() }, sheetContent)
+        }
+    }
+}
+
+
+@Composable
+private fun SheetContent(
+    sheetShape: RoundedCornerShape,
+    sheetBackgroundColor: Color,
+    onBackGroundClicked: () -> Unit,
+    sheetContent: @Composable () -> Unit
+) {
```

Comment:
```suggestion
    shape: RoundedCornerShape,
    backgroundColor: Color,
    onBackGroundClicked: () -> Unit,
    content: @Composable () -> Unit
) {
```

no need to add name of composable in params